docker_test.go

```diff
@@ -795,4 +1019,316 @@ func consume(topics []string, master sarama.Consumer) (chan *sarama.ConsumerMess
 	return consumers, errors
 }
 
-// TODO: Verify in Live Evets API
+func SetZookeeper() *dockertest.Resource {
+	network, err = pool.Client.CreateNetwork(dc.CreateNetworkOptions{Name: "kafka_network"})
```

Comment:
This network is not getting pruned. Can we do something like this: 
```
type ZooKeeperTestT struct {
	network     *dc.Network
	resource    *dockertest.Resource
}
```
and use this later to prune both the network and the resource.